### ISSUE TITLE
The docker refuses to start

### ISSUE THREAD

**Issue by Onotole-s** [docker/for-mac#5619]
- [ х] I have tried with the latest version of Docker Desktop
  - [ х] I have tried disabling enabled experimental features
  - [ x] I have uploaded Diagnostics
  - Diagnostics ID: 22C95F58-DD53-4D18-BDA0-3EE538B3DAB3/20210423111530

### Expected behavior
Docker Desktop started 
### Actual behavior
Docker Desktop stopped
### Information

  - macOS Version: Big Sur 11.2.3
  - Intel chip or Apple chip: Interl
  - Docker Desktop Version: 

### Steps to reproduce the behavior

  1. Very strange problem. OS X reinstallation doesn't help, docker reinstallation doesn't help. 
  2. Docker can't start

**Comment by rreinurm** [comment]
I had similar issue and I managed to purge Docker with script https://github.com/docker/for-mac/issues/5426#issuecomment-791622494

**Comment by dhinojosa** [comment]
I got it to work (Maybe). I moved ~/.kube/config to another backup file name and it worked. All my Kubernetes clusters are down right now intentionally although I still had a config file that points to them. I think somehow it is trying to resolve what Kubernetes clusters you have running, and if it points to a cluster that doesn't exist then Docker never starts.

Hope that works for all of you. Let me know. I am interested.
